core/lib/module/index.ts

```diff
@@ -153,12 +133,29 @@ export class VueStorefrontModule {
   }
 }
 
-export function registerModules (modules: VueStorefrontModule[], context): void {
-  let registeredModules = []
-  modules.forEach(m => registeredModules.push(m.register()))
+function registerModules (modules: VueStorefrontModule[], context): void {
+  modules.forEach(m => m.register())
   Logger.info('VS Modules registration finished.', 'module', {
       succesfulyRegistered: registeredModules.length + ' / ' + modules.length,
       registrationOrder: registeredModules
     }
   )()
 }
+
+function isModuleRegistered(key: string) : boolean {
+  return registeredModules.find(m => m.key === key) ? true : false
```

Comment:
```suggestion
  return registeredModules.some(m => m.key === key)
```
its cleaner that way, because some returns boolean right way